config/crd/bases/cluster.kubernetes-fleet.io_memberclusters.yaml

```diff
@@ -27,7 +27,7 @@ spec:
       name: Age
       type: date
     - jsonPath: .status.properties.kubernetes-fleet\.io/node-count.value
-      name: Node-Count
+      name: NodeCount
```

Comment:
is this not a breaking change?

Reply:
The "Node-Count" is better here as usually k8 is using "-" to split the words.

Could you please fix the API itself instead of the CRD?